proxy/connect/connect.go

```diff
@@ -0,0 +1,186 @@
+package connect
+
+import (
+	"context"
+	"io"
+	"time"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/sirupsen/logrus"
+	"github.com/stripe/veneur/v14/forwardrpc"
+	"github.com/stripe/veneur/v14/proxy/grpcstats"
+	"github.com/stripe/veneur/v14/samplers/metricpb"
+	"github.com/stripe/veneur/v14/scopedstatsd"
+	"google.golang.org/grpc"
+)
+
+type Connect interface {
+	Connect(context.Context, string, DestinationHash) (Destination, error)
+}
+
+var _ Connect = &connect{}
```

Comment:
What does this statement/syntax do? I see you used it a few times. From my understanding, it's not adding anything, but I could be wrong.

Reply:
This ensures that `*connect` implements `Connect`